content/en/docs/getting-started/_index.html

```diff
@@ -0,0 +1,266 @@
+        </li>
+      </ul>
+      <div id="provision-cluster-tabs" class="tab-content">
+        <div class="tab-pane fade show active" id="gcp">
+          <div class="row">
+            <div class="col-sm-4">
+                  <a href="/docs/getting-started/setup/create-cluster/google/"><img src="https://www.pulumi.com/logos/tech/gcp.svg"/></i></a>
+            </div>
+          </div>
+          <h4>Use GKE Terraform Module</h4>
+          <p>1 . Create a <strong>main.tf</strong> file and add the code snippet below.</p>
+{{<highlight javascript>}}
+module "jx" {
+  source  = "jenkins-x/jx/google"
+  version = "1.2.1"
+  # override default parameter values here
+}
+{{</highlight>}}
+
+          <p>This setup uses defaults parameter values.  You can modify them, <a href="/docs/getting-started/setup/create-cluster/gke/#inputs">more details here.</a></p>
+        
+          <p>2 . Provision the cluster using Terraform. The output of this command will include a file <strong>jx-requirements.yml</strong> which<br/>
+          you will use in <strong>Step 3: Install Jenkins X on your cluster</strong> </p>
+{{<highlight bash>}}
+$ terraform init && terraform apply
+{{</highlight>}}
+
+        </div>
+        <div class="tab-pane fade" id="aws">
+        <div class="row">
+          <div class="col-sm-4">
+                <a href="/docs/getting-started/setup/create-cluster/amazon/"> <img src="https://www.pulumi.com/logos/tech/aws.svg" class="mx-auto d-block"/></i></a>
+          </div>
+        </div>
+        <h4>Use the AWS Terraform Module</h4>
+        <p>1 . Create a <strong>main.tf</strong> file and add the code snippet below.</p>
+{{<highlight javascript>}}
+  module "eks-jx" {
+    source  = "jenkins-x/eks-jx/aws"
+    vault_user=""
+  }
+{{</highlight>}}
+This setup uses defaults parameter values. You can modify them, more <a href="docs/getting-started/setup/create-cluster/eks/#inputs">details here</a>.<br>
+<div class="card text-white bg-primary mb-3" style="max-width: 70rem;">
+  <div class="card-header">IAM User</div>
+  <div class="card-body">
+    <h4 class="card-title">Specify existing IAM User</h4>
+    <p class="card-text">You will need to provide a existing IAM user name for Vault. The specified user's access keys are used to authenticate the Vault pod against AWS. The IAM user does not need any permissions attached to it. This Terraform Module creates a new IAM Policy and attaches it to the specified user. <br>
+      For more information refer to <strong><a class="card-link text-white" href="/docs/getting-started/setup/boot/clouds/amazon/#configuring-vault-for-eks">Configuring Vault for EKS</a></strong>.</p>
+  </div>
+</div>
+
+
+          <p>2 . Provision the cluster using Terraform. The output of this command will include a file <strong>jx-requirements.yml</strong> which<br/>
+            you will use in <strong>Step 3: Install Jenkins X on your cluster</strong> </p>
+{{<highlight bash>}}
+$ terraform init && terraform apply
```

Comment:
no '$' to indicate a shell prompt. We should try to be consistent. We don't have it in the rest of the documentation.

Also, I think it is better to split the two lines.

Reply:
This implies that you have `terraform` installed which probably should be mentioned as a prerequisite. Maybe with a link on how to install it.

Reply:
We are also missing the fact that the application default credentials need to be set up via `gcloud auth application-default login`.